symantec_endpoint_security/plugin.spec.yaml

```diff
@@ -816,3 +816,38 @@ actions:
         description: "Agent matching the search"
         type: agent
         required: false
+  blacklist:
+    title: "Blacklist"
+    description: "Blacklist MD5 or SHA256 hashes"
+    input:
+      hashes:
+        title: "Hashes"
+        description: "Hashes (MD5 or SHA256) to add to the blacklist. Note: only one type of hash is allowed at a time"
```

Comment:
Any reason we don't automatically detect the hashes and make the appropriate requests vs requiring the user to make sure they're all the same type?

Reply:
The plugin does that already. SEP is an "one or the other" type of API where you cannot mix MD5 and SHA256 hashes in the same list, hence the description

Reply:
Yeah, I mean. Break the list into md5[] and sha256[] and send each.

Reply:
We could do this in a later version if we need to. As it's not necessary for the workflows but wanted to call this out.

Reply:
Yeah we can revisit options after MVP - getting the solution right may be a bit of work around the UX for the user and the resulting UX in the Symantec console.